src/style-spec/reference/v8.json

```diff
@@ -2971,6 +2973,11 @@
             "android": "6.0.0",
             "ios": "4.0.0",
             "macos": "0.7.0"
+          },
+          "collator": {
+            "js": "0.45.0",
+            "ios": "4.2.0",
+            "macos": "0.9.0"
```

Comment:
@ChrisLoer, I think we should call out the collator syntax separately for each of these comparison operators, since it was introduced after the comparison operators themselves. Not sure if there need to be any changes in docs/pages/style-spec.js.

Reply:
Yeah, this should work and I agree it makes sense to call out separately.